package.json

```diff
@@ -69,7 +69,17 @@
     "typography-plugin-code": "^0.16.20"
   },
   "overrides": {
+    "@enact/docs-utils": {
+      "documentation": {
+        "vue-template-compiler": "^1.0.28"
```

Comment:
this moderate vulnerability seems to come from @enact/docs-utils. Let's wait for a fix from there and not add this override here